packages/features/ee/teams/lib/payments.ts

```diff
@@ -317,3 +348,20 @@ export const updateQuantitySubscriptionFromStripe = async (teamId: number) => {
     console.error(message);
   }
 };
+export const scheduleDebouncedSeatBilling = async () => {
```

Comment:
If this function runs multiple times it will create duplicate tasks. Not really debouncing behavior. We should check if we already have a task Id and push it back by updating that one instead. Alternatively we can remove it if hasn't run and replace it with a new one.